packages/studio/astro.config.mjs

```diff
@@ -1,9 +1,8 @@
-import react from '@astrojs/react'
```

Comment:
Removing this will cause Astro to throw a "React" is not defined error in the users' project. Esp when they don't use react.

Reply:
Yes. I think we can remove that one. It’s not used